apps/web/pages/export.tsx

```diff
@@ -0,0 +1,339 @@
+import {
+  Button,
+  Card,
+  CircularProgress,
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+} from '@usevenice/ui'
+
+import {
+  CheckCircleFilledIcon,
+  CodeIcon,
+  CopyTextIcon,
+  DownloadIcon,
+  SyncIcon,
+} from '@usevenice/ui/icons'
+
+import {useQuery} from '@tanstack/react-query'
+import {twMerge} from 'tailwind-merge'
+import Link from 'next/link'
+import {useEffect, useMemo, useState} from 'react'
+import {ExternalLink} from '../components/ExternalLink'
+import {PageHeader} from '../components/PageHeader'
+import {PageLayout} from '../components/PageLayout'
+import {browserSupabase} from '../contexts/common-contexts'
+
+const PREVIEW_LIMIT = 8
+const DOWNLOAD_LIMIT = 100
+
+export default function Page() {
```

Comment:
@tonyxiao I didn't add `getServerSideProps`—might look into it as a follow-up. Admittedly I'm still not super familiar with next's SSR.